llvm/lib/MC/MCCASObjectV1.cpp

```diff
@@ -1475,7 +1591,11 @@ Error MCCASBuilder::buildFragments() {
         auto Relocs = RelMap.find(&F);
         if (Relocs != RelMap.end())
           AtomRelocs.append(Relocs->second.begin(), Relocs->second.end());
-      }
+        createAddendVector(AtomAddends, ObjectWriter.getAddends()[&F], &F,
+                           Layout);
+      } else
```

Comment:
Right now, if the `RelocLocation == CompileUnit` I still store the Addends in the `SectionRef`, I do not know how I can store them in a separate CAS Block and then reapply them later